index.js

```diff
@@ -0,0 +1,55 @@
+import { loadBooks } from './modules/Save-and-load.js';
+import Book, { addBook, showBook } from './modules/books.js';
+import showTime from './modules/Datetime.js';
+
+let books = [];
+
+window.addEventListener('load', () => {
+  // display the date
+  showTime();
+  // fetch data from local storage
+  books = loadBooks('books');
+  if (books !== null) {
+    books.forEach((element) => {
+      // to be used in the parent component
+      document.getElementById('book-cont').appendChild(showBook(element));
+    });
+  } else {
+    // no books found
+    books = [];
+  }
+});
+
+document.getElementById('form').addEventListener('submit', (e) => {
+  e.preventDefault();
+  let tempBooks = [];
+  tempBooks = loadBooks('books');
+  const newId = tempBooks.length === 0 ? 0 : tempBooks[tempBooks.length - 1].id + 1;
```

Comment:
- [ ] There is an error raised here when I try to add a new book, Please fix that.

![image](https://user-images.githubusercontent.com/68796232/206014163-100c77ab-8fe7-4d90-a33d-2698745974e9.png)

Reply:
hey dear @Omar-Muhamad it's fixed.  💯 
I think that happened when I was clearing code to fix linters because I reformatted the js files.

please check and let me know if that works. 
Kindest regards 🤝